api/disabled_parameters_test.go

```diff
@@ -14,6 +16,165 @@ import (
 	"github.com/algorand/indexer/api/generated/v2"
 )
 
+func TestToDisabledMapConfigFromFile(t *testing.T) {
+	expectedValue := &DisabledMapConfig{Data: map[string]map[string][]string{
+		"/sampleEndpoint": {http.MethodGet: {"p2"}},
+	}}
+
+	configFile := filepath.Join("test_resources", "mock_disabled_map_config.yaml")
+
+	// Nil pointer for openapi3.swagger because we don't want any validation
+	// to be run on the config (they are made up endpoints)
+	loadedConfig, err := MakeDisabledMapConfigFromFile(nil, configFile)
+	require.NoError(t, err)
+
+	require.True(t, reflect.DeepEqual(*expectedValue, *loadedConfig))
```

Comment:
Did you find that `DeepEqual` was required here? I believe testify uses that internally so it wouldn't be required. Also, you can avoid the pointer in expectedValue above so that you don't need to dereference it here. 
```suggestion
	require.NotNil(t, loadedConfig)
	require.Equal(t, expectedValue, *loadedConfig)
```